sites/all/themes/orwell/css/orwell.main.css

```diff
@@ -4428,6 +4423,12 @@ html.js input.form-autocomplete {
   margin: 2rem 0;
 }
 
+.page__files {
+  font-size: 1rem;
+  line-height: 1.8rem;
```

Comment:
Line height should always be relative to itself. Also no use of units is recommended.